OneDrive-Cloud-Player/App.xaml.cs

```diff
@@ -50,7 +53,8 @@ sealed partial class App : Application
         public List<CachedDriveItem> MediaItemList
         {
             get { return mediaItemList; }
-            set {
+            set
+            {
```

Comment:
no change? or for consistency?

Reply:
Most likely a result of the Visual Studio formatter.

Reply:
Looking at it now, we use this formatting elsewhere too.